wisdem/drivetrainse/drive_components.py

```diff
@@ -467,7 +467,7 @@ def setup(self):
         self.add_input("rotor_diameter", 0.0, units="m")
         self.add_input("D_top", 0.0, units="m")
         self.add_input("rho", 0.0, units="kg/m**3")
-        self.add_input("yaw_mass_user", 0.0, units="kg")
+        self.add_input("yaw_mass_user", -1.0, units="kg")
```

Comment:
Don't love a default mass with a negative number.  Seems like you could keep "0.0" here and just check if `m_yaw_user > 0.0` below